guide/C/ch_invest.xml

```diff
@@ -971,56 +980,165 @@ Income
 
-      <para>Notice that the <guilabel>Balance</guilabel> is in the units of
+      <para>Notice that the <guilabel>Balance</guilabel> (last column) is in the units of
       the commodity (AMZN shares) not in currency units. Thus, the balance is
       100 (AMZN units) rather than $2000. This is how it should be.</para>
     </sect2>
 
     <sect2 id="invest-buy-new2">
       <title>Buying New Shares</title>
 
-      <para>The only difference between setting up a new stock purchase
-      versus the setup for preexisting stocks as described in the previous
-      section is that instead of transferring the money used to purchase the
-      stock from the <emphasis>Equity:Opening Balances</emphasis> account, you use your
-      <emphasis>Assets:Bank ABC</emphasis> account.</para>
-
-      <para>Now you will purchase $5000 of IBM stock, with a commission of $100. First
-      step will be to create the stock account for IBM. The existing <emphasis>Expenses:Commissions</emphasis>
-      account will be used. If you wish to track commissions to the individual stock
-      an additional sub-account would be necessary.</para>
-
-      <para>Now for the transaction, on the first transaction line, enter the date of
-      the purchase (eg: Jan. 3 2005), a description (eg: Initial IBM Purchase),
-      Shares will be skipped (to be calculated), Price (eg: $96.60), and Buy ($5000).
-      You do not need to fill in the <guilabel>Shares</guilabel> column, as it will be
-      calculated for you. The next line in the split transaction will be
-      <emphasis>Expenses:Commissions</emphasis> and fill in Buy ($100).
-      The third line will be to transfer from <emphasis>Assets:Bank ABC:Savings</emphasis>,
-      $5100 to balance the transaction.
-      Your IBM Commodity view should now appear like this:</para>
-
-      <screenshot id="invest-setup-portfolio2">
-        <mediaobject>
-           <imageobject role="html">
-             <imagedata fileref="figures/invest_SetupPortfolio2.png" format="PNG"
-                        srccredit="Cristian Marchi" width="510px"></imagedata>
-           </imageobject>
-           <imageobject role="fo">
-             <imagedata fileref="figures/invest_SetupPortfolio2.png" format="PNG"
-                        srccredit="Cristian Marchi"></imagedata>
-           </imageobject>
+      <para>The main difference between setting up a new stock purchase
+        versus the setup for preexisting stocks as described in the previous
+        section is that instead of transferring the money used to purchase the
+        stock from the <emphasis>Equity:Opening Balances</emphasis> account, you
+        transfer from  your <emphasis>Assets:Bank ABC</emphasis> or <emphasis>
+        Assets:Brokerage Account</emphasis> account.
+      </para>
 
-          <textobject>
-            <phrase>Setup Current Portfolio</phrase>
-          </textobject>
+      <sect3 id="invest-buy-com">
+        <title>Handling Commission</title>
```

Comment:
Hi John,
Thanks for reviewing my work.
I don't know if this is a Queen's vs. American English difference. I'm no English expert but both seem OK to me. I'll happily change them all to "Commissions" to be consistent.

I think saying "Commissions and Fees" many times will become annoying to readers.
How about I add a paragraph to the start of the "Handling Commissions and Fees" section saying:

For conciseness, this document will refer to Commissions. Fees may also be payable for some security trades. Unless otherwise stated, fees are handled the same way.